README.md

```diff
@@ -193,6 +135,47 @@ file:path/to/yourproject/file            read directly from a path on disk (any
 registry:yourrepo/yourimage:tag          pull image directly from a registry (no container runtime required)
 ```
 
+#### Default Cataloger Configuration by scan type
+
+##### Image Scanning:
+- alpmdb
+- rpmdb
+- dpkgdb
+- apkdb
+- portage
+- ruby-gemspec
+- python-package
+- php-composer-installed Cataloger
+- javascript-package
+- java
+- go-module-binary
+- dotnet-deps
+
+##### Directory Scanning:
+- alpmdb
+- apkdb
+- dpkgdb
+- portage
+- rpmdb
+- ruby-gemfile
+- python-index
+- python-package
+- php-composer-lock
+- javascript-lock
+- java
+- java-pom
+- go-module-binary
+- go-mod-file
+- rust-cargo-lock
+- dartlang-lock
+- dotnet-deps
+- cocoapods
+- conan
+- hackage
+
+#### Non Default:
+- rust-audit-binary
```

Comment:
Whilst there's renaming going on, we could switch this to `cargo-auditable-binary`?

Since I added syft integration we've been working on making it easier to generate binaries with audit information, and have renamed the project to cargo-auditable (to reflect the new approach which is to simply install the cargo plugin and run `cargo auditable build/install ...` rather than `cargo build/install`.

Reply:
ok I can do that tomorrow

Reply:
Doing it all right now - about to finish the enum work for the catalogers so I'll stick the rename with that